### ISSUE TITLE
Accessibity improvements needed on the Search Result Page

### ISSUE THREAD

**Issue by miguelvaara** [NatLibFi/Skosmos#1039]
![SearchResultPage](https://user-images.githubusercontent.com/50104570/90440835-8f517d80-e0e0-11ea-85c3-0ebb5cd473b6.jpg)
[Make the changes as shown in the illustration (will be up to date before the sprint) and watch the video]

1) In the search result: Tiny symbols before the list items describing the broader, narrower, related concepts and alike should be described aloud in the screen reader (test by using Orca in Linux or NVDA and Narrator in Windows). Use html alt text for the symbol images.

2) Search options: out of the scope

3) The search result page should offer a possibility to tab through interactive items in a logical order (read more in the comments below).

4) The semantic structure of the page should be improved by relevant and comprehensive heading levels (read more in the comments below)

5) Skip link is a shortcut to important parts of the page (content) and makes the access to content quicker. The skip link helps to avoid unnecessary repetition of static page elements. The skip link should be located in the beginning of the body tag (read more in the comments below).

[ListojenSymbolejaEiSanoiteta_NuolellaLiikutaanPomppien_HeadereitaEiOle.mp4.tar.gz](https://github.com/NatLibFi/Skosmos/files/5085078/ListojenSymbolejaEiSanoiteta_NuolellaLiikutaanPomppien_HeadereitaEiOle.mp4.tar.gz)

**Comment by miguelahonen** [comment]
The skip link should become visible when it receives keyboard focus. Maybe hiding it off screen and then to be positioned on screen when it receives keyboard focus?

**Comment by miguelahonen** [comment]
The Page Title element in html (a screen reader reads the Page Title first):
- [service] - [ontology]: [Search result]: [preferred_label] (depending on the language / fi, sv, en)
E.g. "Finto - YSO: Hakutulos: automaatio"

Skip Link refers to the following section of the page:
- Everything (search results) under the header "[N] results for [concept name]". Everything that belongs to the area indicated by the Light bulb symbol

Tab order:
1) Search result header links (preferred_labels)
2) Search area banner
3) Top menu
4) Language selection

Headers levels (level H1):
1) Search result header links (preferred_labels) (Must be hidden):
```
<h1 class="visually-hidden"> ?
.visually-hidden {position: absolute; left: -[something_big]px; overflow: hidden;} ?
```

The Search options section:
- out of the scope

**Comment by miguelahonen** [comment]
All the information for a screen reader should be available in Finnish, Swedish and English

**Comment by osma** [comment]
This is relevant for Skosmos 2 only.

We will make a new accessibility review for Skosmos 3, see #1871

Closing this issue as it is no longer relevant for current development.
